client/components/editor/teaching-elements/Assessment/feedback/FeedbackItem.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <li>
+    <span class="answer-index">Answer {{ index + 1 }}:</span>
+    <template v-if="answer.length">{{ answer }}</template>
+    <i v-else>Answer not added.</i>
+    <quill-editor
+      v-if="isEditing"
+      :content="feedback"
+      :options="quillOptions"
+      @change="$emit('update', $event)">
+    </quill-editor>
+    <div v-else-if="feedback.length" v-html="feedback"></div>
+    <div v-else>
+      <i>Feedback not added.</i>
+    </div>
+  </li>
```

Comment:
I'm not fond of this html structure in general 😕 Although we don't need them (yet) it would still be nice to have wrappers and corresponding css classes for answer & feedback.